modules/snmp/collect.go

```diff
@@ -0,0 +1,44 @@
+package snmp
+
+import (
+	"github.com/gosnmp/gosnmp"
+)
+
+func (s *SNMP) collect() (map[string]int64, error) {
+	collected := make(map[string]int64)
+	var all_oid []string
```

Comment:
Let's use camelCase.

Reply:
done

Reply:
Thanks! If something is done feel free to "Resolve conversation" after pushing a commit(s)